qemu/tests/driver_in_use.py

```diff
@@ -120,8 +121,8 @@ def run_bg_test_sep(sub_type):
     if params["os_type"] == "windows":
         session = vm.wait_for_login(timeout=timeout)
         utils_test.qemu.windrv_verify_running(session, test,
-                                              driver.split()[0])
-        session = utils_test.qemu.setup_win_driver_verifier(session, driver, vm)
+                                              driver_verifier)
+        session = utils_test.qemu.setup_win_driver_verifier(session, driver_verifier, vm)
```

Comment:
This line would be too long, please split it.

Reply:
Okay,thanks a lot.